package.json

```
@@ -10,7 +10,8 @@
     "check": "dprint check",
     "fmt": "dprint fmt",
     "postinstall": "husky && git config --local core.editor cat",
-    "test": "nargo test",
+    "test": "mocha --exit",
```

As you now added a `.mocharc.json` let's be consisten and use it to define for **all** the mocha options so that we don't have to pass any options via the CLI.
```suggestion
    "test": "mocha",
```

---

@YashBit please don't mark conversation as resolved until we agreed on a solution.  
You don't have to follow my suggestion. You may have a good reason to ignore it: in this case please share it so we can discuss